### PR TITLE
Expand async event exception test to cover scenario with single and multiple exceptions thrown

### LINKED ISSUE #364: Add test for async event exception handling with single exception in it
This is a TCK counterpart of https://github.com/eclipse-ee4j/cdi/issues/599

We currently have this test - https://github.com/eclipse-ee4j/cdi-tck/blob/master/impl/src/main/java/org/jboss/cdi/tck/tests/event/observer/async/handlingExceptions/MultipleExceptionsInObserversNotificationTest.java

However, it only asserts case with multiple exceptions. We should add a similar one that has only one exception in it and assert that it is still wrapped in a `CompletionException`.

### PR DESCRIPTION
**Pull request by manovotn**
Fixes #364

### CODE REVIEW

**Comment by manovotn** [comment]
CI failures are there because of current main branch dependencies on staged artifacts.

File: impl/src/main/resources/tck-audit-cdi.xml [hunk 1]
```diff
@@ -5775,16 +5775,17 @@
...
+            <text>If an event is asynchronous, and an exception is thrown by one or more of its notified observers, the
+                `CompletionStage` returned by `fireAsync` will complete exceptionally with
+                `java.util.concurrent.CompletionException` exception as its cause.
             </text>
         </assertion>
 
         <assertion id="b">
             <text>java.util.concurrent.CompletionException contains all exceptions thrown by observers as suppressed exceptions. They can be accessed as an
-                array of throwable with the getSuppressed method.
+                array of `Throwable` with the getSuppressed method.
```
Comment (Ladicek) [inline impl/src/main/resources/tck-audit-cdi.xml:5788]:
```suggestion
                array of `Throwable` with the `getSuppressed` method.
```

File: impl/src/main/resources/tck-audit-cdi.xml [hunk 2]
```diff
@@ -5775,16 +5775,17 @@
...
     </section>
 
-    <section id="async_exception" title="Handling multiple exceptions thrown during an asynchronous event" level="3">
+    <section id="async_exception" title="Handling exceptions thrown during an asynchronous event" level="3">
         <assertion id="a">
-            <text>If an event is asynchronous, and an exception is thrown by any of its notified observers, the CompletionStage returned by fireAsync will
-                complete exceptionally with java.util.concurrent.CompletionException exception.
+            <text>If an event is asynchronous, and an exception is thrown by one or more of its notified observers, the
+                `CompletionStage` returned by `fireAsync` will complete exceptionally with
+                `java.util.concurrent.CompletionException` exception as its cause.
```
Comment (Ladicek) [inline impl/src/main/resources/tck-audit-cdi.xml:5782]:
```suggestion
                `java.util.concurrent.CompletionException`.
```

**Comment by Ladicek** [comment]
Added 2 suggestions to synchronize with the spec text, otherwise LGTM.

**Comment by manovotn** [comment]
@Ladicek your comments are now addressed.

**Review by Ladicek — APPROVED** [review]

**Pull request merged**